AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -764,11 +758,10 @@ - (void)transitionLayoutWithSizeRange:(ASSizeRange)constrainedSize
       }
       
       ASLayout *previousLayout = _layout;
-      [self _applyLayout:newLayout layoutTransition:nil];
+      [self setCalculatedLayout:newLayout];
```

Comment:
Is there still a need to call it `calculatedLayout` internally? Since the ivar is `_layout`, could we do `setLayout:`?